### PR TITLE
Work is finished according to instructions

### PR DESCRIPTION
**Pull request by edlingao**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -65,55 +65,58 @@
...
-}
-.center-text{
+.container{
     position: relative;
-    top: 50%;
-    text-align: center;
-    font-size: 1.2rem;
-    color: white;
-    font-weight:lighter;
+    width: 80%;
```
Comment (zlayabekrija) [inline css/main.css]: Please remove it since it doesn't cover all screen.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great work, just this minor change, please fix it and resubmit review request.
Happy coding 😄 .

File: css/main.css (hunk 1 above)
Comment (edlingao) [reply css/main.css]: Changes are done and everything is correct

**Review by zlayabekrija — APPROVED** [review]
Looks fine 👏 . Thumbs up from my side. 
Good luck with future projects.
Happy coding 😄

**Pull request merged**